packages/vscode-apollo/create-server-symlink.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+const symlinkDir = require("symlink-dir");
+const path = require("path");
+
+symlinkDir("../../apollo-language-server", "./node_modules/")
+  .then(result => {
+    console.log(result);
+  })
+  .catch(err => console.error(err));
```

Comment:
Should this throw on errors? I think if something goes wrong it may not make sense to continue.